Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -572,9 +573,25 @@ class SettingsTableViewController: UITableViewController {
         return section.count
     }
 
-    override func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+    override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        var headerView = SettingsTableSectionHeaderView()
         let section = settings[section]
-        return section.title?.string
+        if let sectionTitle = section.title?.string {
+            headerView.titleLabel.text = sectionTitle
+        }
+
+        return headerView
+    }
+
+    override func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        var height: CGFloat = 13
```

Comment:
What's this 13 for? Is this the little sliver at the top of the mockup with that empty space before the account email? Maybe add a comment here clarifying this value.